templates/cookie-notice.php

```diff
@@ -5,8 +5,7 @@
 
 <div class="cookie-notice" id="cookie-notice" role="banner">
   <div class="cookie-notice__body">
-    <?= __('This website uses cookies to give you the best possible service. By continuing to use the website, you agree to the use of cookies.', Plugin::L10N) ?>
-    <a class="cookie-notice__link" href="<?= apply_filters('core_standards/cookie_notice/policy_link', '/privacy') ?>" target="_self"><?= __('Learn more', Plugin::L10N) ?></a>
+    <?= sprintf(__('To give you the best possible service, this website uses cookies. By continuing to use it, you agree to our <a href="%s" target="_self">privacy policy</a>.', Plugin::L10N), apply_filters('core_standards/cookie_notice/policy_link', '/privacy')); ?>
```

Comment:
The target is superfluous (_self is the default).

```suggestion
    <?= sprintf(__('To give you the best possible service, this website uses cookies. By continuing to use it, you agree to our <a href="%s">privacy policy</a>.', Plugin::L10N), apply_filters('core_standards/cookie_notice/policy_link', '/privacy')); ?>
```

The class has been removed already, but the string translations were not updated yet.

Reply:
Ack! Sorry. Will do.

Reply:
We can remove the class but we still need the styling applied. If we're going this route then we also need to update the CSS. [This block](https://github.com/netzstrategen/wordpress-core-standards/blob/master/assets/styles/cookie-notice.scss#L21-L33) will need to be nested inside the block above it using an `a` element selector.